docs/visual-basic/language-reference/statements/if-then-else-statement.md

```diff
@@ -93,17 +103,19 @@ If condition Then [ statements ] [ Else [ elsestatements ] ]
   
  In the single-line syntax, you can have multiple statements executed as the result of an `If`...`Then` decision. All statements must be on the same line and be separated by colons.  
   
-## Example  
+<a name="multi-line"></a>
+ 
  The following example illustrates the use of the multiple-line syntax of the `If`...`Then`...`Else` statement.  
   
  [!code-vb[VbVbalrStatements#101](../../../visual-basic/language-reference/error-messages/codesnippet/VisualBasic/if-then-else-statement_1.vb)]  
```

Comment:
It might be good here to make this executable -- so wrap it in:
```
Module Multiline
   Public Sub Main()

      Console.WriteLine(message)
   End Sub
End Module
```
Also, instead of "There are " & count & " items." --> $"There are {count} items." (to use an interpolated string)
And also show output